extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -151,13 +158,52 @@ const managers = {};
  */
 let refreshManagerIdCounter = 0;
 
+
+/**
+ * Initializes a RefreshManager under the following conditions:
+ *   2. SRA is not enabled.
+ *   3. The slot is not inside an ad container (with the exception of
+ *      carousel and sticky ad).
+ *   4. The publisher has provided an appropriate refresh interval (>= 30s).
+ *
+ * @param {!./amp-a4a.AmpA4A} a4a
+ * @param {boolean} useSra
+ * @return {?RefreshManager}
+ */
+export function getRefreshManagerForDoubleclickIfEligible(a4a, useSra) {
```

Comment:
Not crazy about having doubleclick specific requirements within refresh manager.  Anyway we could pass the predicate?

export function getRefreshManager(a4a, opt_predicate) {
   if (getRefreshIntervalFromPageOrElement == invalid) {
      user().warn(...);
      return null;
   }
   if (opt_predicate && !opt_predicate()) {
      user().warn(...);
      return null;
   }
   return new Refresh...
}